tasks/skylab/H5adUtils.wdl

```diff
@@ -317,7 +317,7 @@ task JoinMultiomeBarcodes {
     echo "Starting bgzip"
     bgzip "~{atac_fragment_base}.sorted.tsv"
     echo "Starting tabix"
-    tabix -s 1 -b 2 -e 3 "~{atac_fragment_base}.sorted.tsv.gz"
+    tabix -s 1 -b 2 -e 3 -C "~{atac_fragment_base}.sorted.tsv.gz"
```

Comment:
@khajoue2 Do we know if this is still compatible with ArchR import? We only added tabix in to make the outputs compatible, so if it isn't directly compatible, we might want to flag it to Multiome group.

Reply:
This information wasn't clearly specified in ArchR's public documentation, we can do some downstream check.

Reply:
We can make a note "If incompatible, explore alternative solutions e.g., chromosome splitting."

Reply:
LGTM!